src/main/java/it/pagopa/transactions/commands/handlers/TransactionRequestAuthorizationHandlerCommon.java

```diff
@@ -337,32 +337,59 @@ private Mono<AuthorizationOutput> invokeNpgConfirmPayment(
                                             );
                                             boolean isContextualOnboarding = authorizationData
                                                     .isWalletPaymentWithContextualOnboarding();
+                                            boolean isCheckoutWalletPayment = (clientId
+                                                    .equals(Transaction.ClientId.CHECKOUT.toString()) ||
+                                                    clientId.equals(Transaction.ClientId.CHECKOUT_CART.toString()))
```

Comment:
I think we have to add also WISP_REDIRECT here as client id. maybe this check can be changed to `!clientId.equals(IO)`